zulip/integrations/bridge_with_irc/irc_mirror_backend.py

```diff
@@ -35,15 +36,18 @@ def connect(self, *args, **kwargs):
         self.reactor.loop.run_until_complete(
             self.connection.connect(*args, **kwargs)
         )
-        print("Connected to IRC server.")
+        print("Connected to IRC server. The bot has started.")
```

Comment:
The message that @showell suggested in czo was different, wasn't it. Though I am fine with this one as well.

Reply:
Changed the message.